tests/ci_build/conda_env/linux_sycl_test.yml

```diff
@@ -0,0 +1,20 @@
+name: linux_sycl_test
+channels:
+- conda-forge
+- intel
+dependencies:
+- python=3.8
```

Comment:
should be it 3.8 or higher?

Reply:
Have used the same config, as is already used [for cpu](https://github.com/dmlc/xgboost/blob/fedd9674c8441a26b94f9d4061b11a1be4cb0e7c/tests/ci_build/conda_env/linux_cpu_test.yml#L5C1-L5C1)